casper/src/main/scala/coop/rchain/casper/BlockCreator.scala

```diff
@@ -156,15 +158,17 @@ object BlockCreator {
       maxBlockNumber: Long,
       shardId: String,
       version: Long,
-      now: Long
+      now: Long,
+      span: Span[F]
   ): F[CreateBlockStatus] =
     InterpreterUtil
       .computeDeploysCheckpoint[F](
         parents,
         deploys,
         dag,
         runtimeManager,
-        now
+        now,
+        span
```

Comment:
One day I'm going to take rchain and ReaderT-pattern-the-hell-out-of-it... :joy: